src/plugin/context.rs

```diff
@@ -880,4 +883,28 @@ impl RapierContext {
             )
         });
     }
+
+    /// Computes the angle between the 2 bodies
+    /// referenced by the underlying [`ImpulseJoint`] of the given `entity`,
+    /// according to its [`RevoluteJoint`] description.
+    ///
+    /// Parameter `entity` should have a [`ImpulseJoint`] or [`MultibodyJoint`] component with a [`JointDescription::RevoluteJoint`] variant as `data`.
+    pub fn angle_for_entity_impulse_revolute_joint(&self, entity: Entity) -> Option<f32> {
```

Comment:
Perhaps this could be named `impulse_revolute_joint_angle` (even if it takes an `Entity`)?

Reply:
My objective was to insist on the requisite for given entity to be a `ImpulseJoint`, but I'm ok with shortening it.

Looking at this makes me realize the documentation comment is wrong, I couldn't make this work with multibodyjoint: as multibodyjoints have multiple bodies, the same API cannot really be applied (if I understand correctly)

Reply:
done :)